lib/ical_filter_proxy/alarm_trigger.rb

```diff
@@ -0,0 +1,32 @@
+module IcalFilterProxy
+  class AlarmTrigger
+
+    attr_accessor :alarm_trigger
+
+    def initialize(alarm_trigger)
+      self.alarm_trigger = alarm_trigger
```

Comment:
Why not compile with `generate_iso_format` in the initialiser? Saves us having an additional get_trigger method.

Reply:
Yes agree, done.